src/back/auth/src/repo/table_session.cpp

```diff
@@ -75,8 +75,21 @@ model::Session Session::GetById(const std::string& id, bool isActive)
 	return items.front();
 }
 
+const storages::postgres::Query kInactivateSessionsWithUserId{
+	"UPDATE session SET active = false "
```

Comment:
Если сессия не активна, то и токены не нужны. Только место занимают. 
```suggestion
	"UPDATE session SET active=false, token='', accessToken='', refreshToken='', idToken='' "
```

Reply:
Поправил.